cmd/install.go

```diff
@@ -914,79 +935,162 @@ func installDupeDetection(ctx context.Context, config *configs.Config) (err erro
 
 	os.Setenv("DUPEDETECTIONCONFIGPATH", ddConfigPath)
 
-	if err = installDDImgServer(ctx, config); err != nil {
+	if err = íntallAppService(ctx, "dd-img-server", config); err != nil {
 		return err
 	}
 
 	log.WithContext(ctx).Info("Installing DupeDetection finished successfully")
 	return nil
 }
 
-func installDDImgServer(ctx context.Context, config *configs.Config) error {
-	log.WithContext(ctx).Info("Installing DupeDetection image server")
-
-	ddImgServerServiceFile := "dd_img_server.service"
-	ddImgServerServiceDir := "/etc/systemd/system"
-	ddImgServerServiceFilePath := filepath.Join(ddImgServerServiceDir, ddImgServerServiceFile)
-	ddImgServerServiceTempFilePath := filepath.Join(config.PastelExecDir, ddImgServerServiceFile)
-
-	ddImgServerStartFile := "start_dd_img_server.sh"
-	ddImgServerStartDir := filepath.Join(config.PastelExecDir, constants.DupeDetectionSubFolder)
-	ddImgServerStartFilePath := filepath.Join(ddImgServerStartDir, ddImgServerStartFile)
-	ddBaseDir := filepath.Join(config.Configurer.DefaultHomeDir(), constants.DupeDetectionServiceDir)
-	ddImgServerWorkDir := filepath.Join(ddBaseDir, "img_server")
-
-	// create service file
-	if ddImgServerDaemonScript, err := utils.GetServiceConfig("dd_img_server", configs.DDImgServerService,
-		&configs.DDImgServerServiceScript{
-			DDImgServerStartScript: ddImgServerStartFilePath,
-		}); err != nil {
-		log.WithContext(ctx).WithError(err).Error("unable to create content of dd_img_server file")
-		return fmt.Errorf("unable to create content of dd_img_server file - err: %s", err)
-	} else if err := utils.CreateAndWrite(ctx, config.Force, ddImgServerServiceTempFilePath, ddImgServerDaemonScript); err != nil {
+func íntallAppService(ctx context.Context, appName string, config *configs.Config) error {
+
+	log.WithContext(ctx).Info("Installing " + appName + " as service")
+
+	var systemdFile string
+	var err error
+	var execCmd, execPath, execUser, workDir string
+
+	// Service file - will be installed at /etc/systemd/system
+	appServiceFileName := appName + ".service"
+	systemdDir := "/etc/systemd/system"
```

Comment:
nit: should make it as constants in constants package.

Reply:
appServiceFileName - this one should have prefix to avoid conflict with other systemd service.